PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,11 +21,9 @@ open. -->
 statements. The basic format is a bulleted list with the name of the subpackage
 and the release note for this PR. For example:
 
-* solvers
-  * Added a new solver for logarithmic equations.
-
-* functions
-  * Fixed a bug with log of integers.
```

Comment:
I don't think that the existing entries should be deleted though

Reply:
* solvers
  * Added a new solver for logarithmic equations.
* functions
  * Fixed a bug with log of integers. Formerly, `log(-x)` incorrectly gave `-log(x)`.
* physics.units
  * Corrected a semantical error in the conversion between volt and statvolt, reporting the volt as larger than the statvolt.

these examples might be fine but in second example `Now log(-x) will remain as log(-x) unless x is an explicit number whose sign can be computed (for example log(-3) evaluates to log(3) + I*pi).` what about this condition should we also add this one ?